File: app/src/main/java/rango/tool/androidtool/java/Bob.java
package rango.tool.androidtool.java;

import rango.tool.androidtool.java.Person;

public class Bob extends Person {

    public static void log() {
        System.out.println("Bob");
    }

    @Override
    public String getName() {
        return super.getName();
    }

    @Override
    void setName(String name) {
        super.setName(name);
    }
}

File: app/src/main/java/rango/kotlin/mytest/show/Bob.kt
package rango.kotlin.mytest.show

import rango.kotlin.mytest.share.Person

class Bob(age: Int) : Person("Bob", age) {

    constructor(isMarried: Boolean, age: Int) : this(age) {
        println("4")
    }

    init {
        println("5---age = $age")
    }

    override fun run(speed: Int) {
        super.run(speed)
    }
}